### DISCUSSION TITLE
Github/Gitlab only is a deal breaker.

### DISCUSSION THREAD

**Discussion by kendofriendo** [sveltia/sveltia-cms#185] [General]
I'm not hosting anything on through those sites so I'll keep this project on my radar but I can't use it.

**Comment by kyoshino** [comment]
You can still use Sveltia CMS with your [local repository](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#working-with-a-local-git-repository) and Azure DevOps can _probably_ be supported, but we don’t plan to support Bitbucket, Gitea/Forgejo and Git Gateway as mentioned in the [README](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#features-not-to-be-implemented). Please look for other solutions if Sveltia CMS doesn’t meet your needs.

**Reply by kyoshino** [reply]
And you can host your site anywhere, not just GitHub Pages an GitLab Pages. Most of my clients use Cloudflare Pages.
